Runtime/Scripts/ExampleRest.cs

```diff
@@ -79,7 +79,11 @@ async void GetEdgeConnection()
 
     private void HandleFindCloudlet(EdgeEventsStatus status, FindCloudletEvent fcEvent)
     {
-        throw new NotImplementedException();
+        print("NewFindCloudlet triggered status is " + status.status + ", Trigger" + fcEvent.trigger);
+        if(fcEvent.newCloudlet != null)
+        {
+            print("New Cloudlet FQDN: " + fcEvent.newCloudlet.Fqdn);
+        }
```

Comment:
Should also check for errors in this handler

Reply:
Would something like this be sufficient?
``` 
private void HandleFindCloudlet(EdgeEventsStatus edgeEventstatus, FindCloudletEvent fcEvent)
    {
        print("NewFindCloudlet triggered status is " + edgeEventstatus.status + ", Trigger" + fcEvent.trigger);
        if(fcEvent.newCloudlet != null)
        {
            print("New Cloudlet FQDN: " + fcEvent.newCloudlet.Fqdn);
        }
        if(edgeEventstatus.status == Status.error)
        {
            print("Error received: "+ edgeEventstatus.error_msg);
        }
    }
```